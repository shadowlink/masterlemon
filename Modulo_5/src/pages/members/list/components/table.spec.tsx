import * as React from "react";
import { shallow } from "enzyme";
import { Table } from "./table";
import { Member } from "../viewModel";

describe("members/list/components/table tests", () => {
  it("should render table", () => {
    const members: Member[] = [
      {
        avatarUrl: "url",
        id: 0,
        name: "felipe"
      }
    ];

    // Act
    const component = shallow(<Table members={members} />);

    // Assert
    expect(component).toMatchSnapshot();
  });
});
