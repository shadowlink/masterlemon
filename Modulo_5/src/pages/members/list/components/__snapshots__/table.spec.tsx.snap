// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`members/list/components/table tests should render table 1`] = `
<table
  className="table table-striped"
>
  <Component />
  <Component
    members={
      Array [
        Object {
          "avatarUrl": "url",
          "id": 0,
          "name": "felipe",
        },
      ]
    }
  />
</table>
`;
